deploy/installer/generated/ibm-block-csi-operator.yaml

```diff
@@ -4,6 +4,13 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
```

Comment:
what about the `Deployment` (`spec.template.metadata.annotations`) changes?

Reply:
done (thanks)
arch and memory changes were added too